extra/man/alacritty.5.scd

```diff
@@ -689,75 +689,123 @@ https://docs.rs/winit/\*/winit/keyboard/enum.Key.html
 
 	*action*
 
+		*Paste*
+			Paste contents of system clipboard.
+		*Copy*
+			Store current selection into clipboard.
+		*IncreaseFontSize*
+			Increase font size.
+		*DecreaseFontSize*
+			Decrease font size.
+		*ResetFontSize*
+			Reset font size to the config value.
+		*ScrollPageUp*
+			Scroll exactly one page up.
+		*ScrollPageDown*
+			Scroll exactly one page down.
+		*ScrollHalfPageUp*
+			Scroll half a page up.
+		*ScrollHalfPageDown*
+			Scroll half a page down.
+		*ScrollLineUp*
+			Scroll one line up.
+		*ScrollLineDown*
+			Scroll one line down.
+		*ScrollToTop*
+			Scroll all the way to the top.
+		*ScrollToBottom*
+			Scroll all the way to the bottom.
+		*ClearHistory*
+			Clear the display buffer(s) to remove history.
+		*Hide*
+			Hide the Alacritty window.
+		*Minimize*
+			Minimize the Alacritty window.
+		*Quit*
+			Quit Alacritty.
+		*ClearLogNotice*
+			Clear warning and error notices.
+		*SpawnNewInstance*
+			Spawn a new instance of Alacritty.
+		*CreateNewWindow*
+			Create a new Alacritty window.
+		*ToggleFullscreen*
+			Toggle fullscreen.
+		*ToggleMaximized*
+			Toggle maximized.
+		*ClearSelection*
+			Clear active selection.
+		*ToggleViMode*
+			Toggle vi mode.
+		*ReceiveChar*
+			Allow receiving char input.
+		*SearchForward*
+			Start a forward buffer search.
+		*SearchBackward*
+			Start a backward buffer search.
+		*None*
+			No action.
```

Comment:
Not sure whether we should mention this one here as well.

Reply:
For sure, I'd actually move `ReceiveChar` and `None` to the top probably?